en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md

```diff
@@ -0,0 +1,262 @@
+# Amazon Lambda Connector Example
+
+AWS Lambda is a compute service that lets you run code without provisioning or managing servers. AWS Lambda executes your code only when needed and scales automatically, from a few requests per day to thousands per second. 
+The following operations allow you to work with functions in Amazon Lambda.
+
+Given below is a sample scenario that demonstrates how to create a Amazon Lambda Function in the AWS Lambda Service using the WSO2 Amazone Lambda Connector.
+
+## What you'll build
+To use the Amazon Lambda connector, add the <amazonlambda.init> element in your configuration before carrying out any Amazon Lambda operations. This Amazon Lambda configuration authenticates with Amazon Lambda by specifying the AWS access key ID and secret access key ID, which are used for every operation. The signature is used with every request and thus differs based on the request the user makes.
+
+**initialize the connector**
+
+You can add the parameter as follows in the init operation:
+
+````xml
+<amazonlambda.init>
+    <region>{$ctx:region}</region>
+    <secretAccessKey>{$ctx:secretAccessKey}</secretAccessKey>
+    <accessKeyId>{$ctx:accessKeyId}</accessKeyId>
+    <blocking>{$ctx:blocking}</blocking>
+</amazonlambda.init>
+````
+**createFunction**
+
+The createFunction operation creates a Lambda function. To create a function, you need a deployment package and an execution role. The deployment package contains your function code. The execution role grants the function permission to use AWS services.
+
+You can add the parameter as follows in the createFunction operation:
+
+````xml
+<amazonlambda.createFunction>
+    <apiVersionCreateFunction>{$ctx:apiVersionCreateFunction}</apiVersionCreateFunction>
+    <functionName>{$ctx:functionName}</functionName>
+    <functionDescription>{$ctx:functionDescription}</functionDescription>
+    <s3Bucket>{$ctx:s3Bucket}</s3Bucket>
+    <s3Key>{$ctx:s3Key}</s3Key>
+    <s3ObjectVersion>{$ctx:s3ObjectVersion}</s3ObjectVersion>
+    <zipFile>{$ctx:zipFile}</zipFile>
+    <targetArn>{$ctx:targetArn}</targetArn>
+    <environmentVariables>{$ctx:environmentVariables}</environmentVariables>
+    <handler>{$ctx:handler}</handler>
+    <kmsKeyArn>{$ctx:kmsKeyArn}</kmsKeyArn>
+    <layers>{$ctx:layers}</layers>
+    <memorySize>{$ctx:memorySize}</memorySize>
+    <publish>{$ctx:publish}</publish>
+    <role>{$ctx:role}</role>
+    <runtime>{$ctx:runtime}</runtime>
+    <tags>{$ctx:tags}</tags>
+    <timeout>{$ctx:timeout}</timeout>
+    <mode>{$ctx:mode}</mode>
+    <securityGroupIds>{$ctx:securityGroupIds}</securityGroupIds>
+    <subnetIds>{$ctx:subnetIds}</subnetIds>
+</amazonlambda.createFunction>
+````
+The following diagram illustrates all the required functionality of the Amazone Lambda Service that you are going to build.
+
+<img src="/assets/img/connectors/AmazonLambdaConnectorSample.png" title="Amazon Lambda Connector" width="800" alt="Amazon Lambda Connector"/>
```

Comment:
Give some explanation of the diagram as well.

Reply:
Added small description about the whole sample.